qa/deepsea/nodes/1node.yaml

```diff
@@ -1,2 +1,2 @@
 roles:
-- [client.salt_master]
+  - [client.salt_master, mon.a, mgr.x, osd.0]
```

Comment:
Please make this change inside `deepsea/tier4/drive-replacement` instead of in `qa/deepsea`. In other words, `qa/suites/deepsea/tier4/drive-replacement/cluster/1node.yaml` should not be a symlink.

See how it is done in the other tests (`cluster/roles.yaml` is typically not a symlink).